docs/api-reference.md

```diff
@@ -3,25 +3,623 @@ id: api-reference
+| [count](#count)             | Returns the count of items that meet a condition. |
+| [insert](#insert)           | Inserts an item into a table.                     |
+| [update](#update)           | Searches for data items and updates them.         |
+| [delete](#delete)           | Deletes data items.                               |
+
+### defineTable
+
+Defines a new table within the store.
+
+#### Arguments
+
+| Argument Name    | Description                                         |
+| ---------------- | --------------------------------------------------- |
+| tableName        | The new table's name.                               |
+| tableDescription | An object that describes the new table's structure. |
+
+#### Example
+
+```js
+Init: async store => {
+  await store.defineTable('Stories', {
+    indexes: { id: 'string', type: 'string' },
+    fields: [
+      'title',
+      'text',
+      'link',
+      'commentCount',
+      'votes',
+      'createdAt',
+      'createdBy',
+      'createdByName'
+    ]
+  })
+```
+
+### find
+
+Searches for data items that based on the specified expression.
+
+#### Arguments
+
+| Argument Name    | Description                                 |
+| ---------------- | ------------------------------------------- |
+| tableName        | A table name.                               |
+| searchExpression | An object that defines a search expression. |
+| fieldList        | A list of fields to fetch.                  |
+| sort             | The sort order.                             |
+| skip             | A number of data items to skip.             |
+| limit            | The maximum number of data items to fetch.  |
+
+#### Example
+
+```js
+const getStories = async (type, store, { first, offset }) => {
+  await store.waitEventCausalConsistency()
+  const search = type && type.constructor === String ? { type } : {}
+  const skip = first || 0
+  const stories = await store.find(
+    'Stories',
+    search,
+    null,
+    { createdAt: -1 },
+    skip,
+    skip + offset
+  )
+  return Array.isArray(stories) ? stories : []
+}
+```
+
+### findOne
+
+Searches for a single data item based on the specified expression.
+
+#### Arguments
+
+| Argument Name    | Description                                 |
+| ---------------- | ------------------------------------------- |
+| tableName        | A table name.                               |
+| searchExpression | An object that defines a search expression. |
+| fieldList        | A list of fields to fetch.                  |
+
+#### Example
+
+```js
+[STORY_UPVOTED]: async (store, { aggregateId, payload: { userId } }) => {
+  const story = await store.findOne(
+    'Stories',
+    { id: aggregateId },
+    { votes: 1 }
+  )
+  await store.update(
+    'Stories',
+    { id: aggregateId },
+    { $set: { votes: story.votes.concat(userId) } }
+  )
+},
+```
+
+### count
+
+Returns the count of items that meet the specified condition.
```

Comment:
Why 'the count'? 
```suggestion
Returns the number of items that meet the specified condition.
```

Reply:
Do I have to correct the same things everywhere, or are you capable of going through your own text and correcting it yourself?